### ISSUE TITLE
Fixing command in Makefile

### ISSUE THREAD

**Issue by Sddilora** [octoposprime/op-be-logging#24]
fix this part
```bash
## Run:
local-run: ## Run in Local for Development
	LOCAL = true && export LOCAL && go run cmd/service/main.go
```
as
```bash
## Run:
local-run: ## Run in Local for Development
	LOCAL=true go run cmd/service/main.go
```
in Makefile

**Comment by Maraza25** [comment]
closes #24

**Comment by Sddilora** [comment]
Close the issue after pull request merged succesfully 
@Maraza25
